syntax = "proto3";

package transaction_verification;

service TransactionVerificationService {
  rpc VerifyTransaction (TransactionVerificationRequest) returns (TransactionVerificationResponse);
}

message TransactionVerificationRequest {
  string order_json = 1;
  string vector_clock_json = 2;
}

message TransactionVerificationResponse {
  string response_json = 1;
  string vector_clock_json = 2;
}
